.github/scripts/bin_diff.ps1

```diff
@@ -0,0 +1,258 @@
+################################################################################
```

Comment:
is this script based on some sample code? Did it have a license, can we link to it?

Reply:
Added source.